build/karma.conf.babel.js

```diff
@@ -42,23 +42,19 @@ module.exports = (karmaConfig) => {
     },
     webpack: {
       devtool: 'cheap-module-source-map',
-      module: {
-        ...webpackConfig.module,
+      module: Object.assign({}, webpackConfig.module, {
```

Comment:
Would it make sense to use a spread operator here? I _believe_ they're in node6.